src/commands/up.rs

```diff
@@ -154,16 +155,33 @@ impl UpCommand {
             _ => unreachable!(),
         };
 
+        let mut wasmtime_config = wasmtime::Config::default();
```

Comment:
Let's still keep this in a separate function, even if that's a smaller function than it was before - the `run` function is getting very long and I'm finding it hard to navigate.